draft-zf-community-dev-fund-2-proposal.rst

```diff
@@ -0,0 +1,486 @@
+
+The slices are described in more detail below. The fund flow will be 
+implemented at the consensus-rule layer, by sending the corresponding ZEC to 
+the designated address(es) for each block. This Dev Fund will end at block 
+height 3566400 (unless extended/modified by a future ZIP).
+
+
+BP slice (Bootstrap Project)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This slice of the Dev Fund will flow as charitable contributions from the 
+Zcash Community to the Bootstrap Project, the newly formed parent organization 
+to the Electric Coin Company. The Bootstrap Project is organized for exempt 
+educational, charitable, and scientific purposes in compliance with Section 
+501(c)(3), including but not limited to furthering education, information, 
+resources, advocacy, support, community, and research relating to 
+cryptocurrency and privacy, including Zcash. This slice will be used at the 
+discretion of the Bootstrap Project for any purpose within its mandate to 
+support financial privacy and the Zcash platform as permitted under Section 
+501(c)(3). The BP slice will be treated as a charitable contribution from the 
+Community to support these educational, charitable, and scientific purposes.
+
+
+ZF slice (Zcash Foundation)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This slice of the Dev Fund will flow as charitable contributions from the 
+Zcash Community to ZF, to be used at its discretion for any purpose within its 
+mandate to support financial privacy and the Zcash platform, including: 
+development, education, supporting community communication online and via 
+events, gathering community sentiment, and awarding external grants for all of 
+the above, subject to the requirements of Section 501(c)(3). The ZF slice will 
+be treated as a charitable contribution from the Community to support these 
+educational, charitable, and scientific purposes.
+
+
+ZCG slice (Zcash Community Grants)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This slice of the Dev Fund is intended to fund independent teams entering the
+Zcash ecosystem, to perform major ongoing development (or other work) for the
+public good of the Zcash ecosystem, to the extent that such teams are 
+Available and effective.
+
+The funds SHALL be received and administered by FPF. FPF MUST disburse them 
+for "Zcash Community Grants" and expenses reasonably related to the 
+administration of Zcash Community Grants, but subject to the following 
+additional constraints:
+
+1. These funds MUST only be used to issue Zcash Community Grants to external 
+   parties that are independent of FPF or to Autonomous Entities that operate 
+   under the FPF umbrella, and to pay for expenses reasonably related to 
+   the administration of Zcash Community Grants. They MUST NOT be used by FPF 
+   for its internal operations and direct expenses not related to 
+   administration of Zcash Community Grants. Additionally, BP, ECC, and ZF are
+   ineligible to receive Major Grants.
```

Comment:
This is unclear as to ECC's eligibility to receive Major Grants after their funding stream would expire under the proposal. I believe the implication of ECC's stated desire to move to a decentralized grant-based funding model is that it *not* be excluded from Major Grants (especially not after the expiry of its funding stream under this proposal). I will ask Josh Swihart about this.

Reply:
I have confirmed with Josh that our position is that we would accept Major Grants, and we consider being excluded from them to be a deficiency of the proposal.

Reply:
Updated to amend the eligibility criteria (and to reflect the name change from Major Grants to Zcash Community Grants).